ReactWindows/ReactNative.Net46/Views/Text/ReactTextShadowNode.cs

```diff
@@ -213,8 +213,8 @@ private static YogaSize MeasureText(ReactTextShadowNode textNode, YogaNode node,
                 var normalizedHeight = YogaConstants.IsUndefined(height) ? double.PositiveInfinity : height;
                 textBlock.Measure(new Size(normalizedWidth, normalizedHeight));
                 return MeasureOutput.Make(
-                    (float)textBlock.DesiredSize.Width,
-                    (float)textBlock.DesiredSize.Height);
+                    (float)Math.Ceiling(textBlock.DesiredSize.Width),
```

Comment:
>Ceiling [](start = 32, length = 7)

That's strange that cropping occurs on WPF and not on UWP.  We were going to use the same technique for UWP, but fixed the rounding issue in Yoga.  [https://github.com/facebook/yoga/pull/300](https://github.com/facebook/yoga/pull/300)